hsl_builder/Text.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from .Base import BaseElement
+from .Elements.Actionable import Actionable
+
+class Text(BaseElement):
+    """
+    Create simple text message HSL
+    """
+    def __init__(self,text):
+        super().__init__(text,'TEXT')
+        self.quick_replies: List[Actionable] = [] #TODO subclass actionables for qr
```

Comment:
This TODO needs to be fixed?

Reply:
removing this todo since there isn't much benefit to sublassing actionables as qucik replies.